parsons/targetsmart/targetsmart_automation.py

```diff
@@ -9,75 +16,109 @@
-                A table object with the required columns. (Required columns provided be TargetSmart)
+                A table object with the required columns. Each workflow type
+                requires the input file to meet the requirements provided by
+                TargetSmart. You can locate the input and output layouts for
+                your available workflows using the My TargetSmart Automation
+                console.
             job_type: str
-                The match job type. **This is case sensitive.** (Match job names provided by TargetSmart)
+                The workflow name to execute. **This is case sensitive.**. You
+                can locate the workflow names and other information by visiting
+                the Automation console in My TargetSmart.
             job_name: str
-                Optional job name.
+                Optional job execution name.
             emails: list
                 A list of emails that will received status notifications. This
                 is useful in debugging failed jobs.
             call_back: str
                 A callback url to which the status will be posted. See
-                `TargetSmart documentation <https://docs.targetsmart.com/developers/automation/index.html#http-callback>`_
+                `TargetSmart documentation <https://docs.targetsmart.com/my_tsmart/automation/developer.html>`_
                 for more details.
             remove_files: boolean
                 Remove the configuration, file to be matched and matched file from
-                the TargetSmart FTP upon completion or failure of match.
-        """ # noqa: E501,E261
+                the TargetSmart SFTP upon completion or failure of match.
+
+        """  # noqa: E501,E261
 
         # Generate a match job
         job_name = job_name or str(uuid.uuid1())
 
         try:
             # Upload table
-            self.sftp.put_file(table.to_csv(), f'{self.sftp_dir}/{job_name}_input.csv')
-            logger.info(f'Table with {table.num_rows} rows uploaded to TargetSmart.')
+            self.sftp.put_file(table.to_csv(), f"{self.sftp_dir}/{job_name}_input.csv")
+            logger.info(f"Table with {table.num_rows} rows uploaded to TargetSmart.")
 
             # Create/upload XML configuration
-            xml = self.create_job_xml(job_type, job_name, emails=emails,
-                                      status_key=job_name, call_back=call_back)
-            self.sftp.put_file(xml, f'{self.sftp_dir}/{job_name}.job.xml')
-            logger.info('Match configuration uploaded to TargetSmart.')
+            xml = self.create_job_xml(
+                job_type,
+                job_name,
+                emails=emails,
+                status_key=job_name,
+                call_back=call_back,
+            )
+            self.sftp.put_file(xml, f"{self.sftp_dir}/{job_name}.job.xml")
+            logger.info("Match configuration uploaded to TargetSmart.")
```

Comment:
`logger.info(f"Match configuration {job_type} uploaded to TargetSmart.")`

Might be nice to log the `job_type`.